### PR TITLE
openlabcmd py3 support

### PR DESCRIPTION
**Pull request by wangxiyuan**
1. Remove `commands` lib usage.
2. Add `enum` requirement only for py2.

### CODE REVIEW

File: openlabcmd/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-pbr>=1.3
-PyYAML>=5.1
+enum;python_version=='2.7'
```
Comment (kiwik) [inline openlabcmd/requirements.txt:1]: Question: The update only is supported by py3,  or both py2 and py3? I find the default python version is python 2.7 in OpenLab CI host (zuul and nodepool), so we need to make sure openlabcmd can work on python 2.7

Comment (wangxiyuan) [reply openlabcmd/requirements.txt:1]: openlabcmd support both py2 and py3.

`enum` in py3 is a built-in library, so we only install it in py2.

Comment (kiwik) [reply openlabcmd/requirements.txt:1]: OK, thank you

**Review by kiwik — APPROVED** [review]
LGTM

**Pull request merged**